### PR TITLE
Step 1 of the sample app

### PR DESCRIPTION
**Pull request by nsiddarthreddy**
This is a step -1 of the sample project.

First iteration
App must live in a git repository in github.com
Model with 3 fields. (stuff saved database)
Bellow the form in the page, there must be list of the already submitted entries to the database.
App must have unittests
Create Pull Request / Get code reviewed by team mate / Merge Pull Request

### CODE REVIEW

File: levenshtein/views.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+from django.shortcuts import render
+from django.views.generic import TemplateView
+
+from levenshtein.models import Levenshtein
+from levenshtein.forms import LevenshteinForm
+
+def index(request):
```
Comment (diegobz) [inline levenshtein/views.py]: Let's use a class based view instead.

Comment (nsiddarthreddy) [inline levenshtein/views.py]: ok i can change that.

File: levenshtein/views.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        if form.is_valid():
+            levenshtein = form.save()
+            levenshtein.levenshtein_value = levenshteinDistance(levenshtein.text_1, levenshtein.text_2)
+            levenshtein.save()
+    else:
+        form = LevenshteinForm()
+
+    return render(request, 'levenshtein/index.html', {'form': form, 'levenshtein_objects':levenshtein_objects})
+
+def levenshteinDistance(s1,s2):
```
Comment (diegobz) [inline levenshtein/views.py]: This function should live in a `utils.py` or something, as this is actually a helper function. Generally only views should be in the `views.py`.

Comment (diegobz) [inline levenshtein/views.py]: Also `(s1,s2)` should be turned into `(s1, s2)`. Please enable a PEP8 linter for your IDE/Editor, so you can get help to catch this things more easily.

File: levenshtein/tests.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+from django.test import TestCase
+from levenshtein.models import Levenshtein
+from levenshtein.forms import LevenshteinForm
+
+class LevenshteinTestCase(TestCase):
+    def setUp(self):
+        Levenshtein.objects.create(text_1="kitten", text_2="sitten")
```
Comment (diegobz) [inline levenshtein/tests.py]: Why do you need to create these 2 objects?

**Comment by diegobz** [comment]
In general good stuff... basically minor code styling comments. :) :+1:

**Comment by nsiddarthreddy** [comment]
I made all the changes you suggested , please check it out. Usually when you ask for changes in a pull request do i need to create a new pull request or just commit the changes and commit here?

**Comment by diegobz** [comment]
Nice! Just remove the `db.sqlite3` from the commit and add it to the .gitignore file and you are done. :)

**Comment by nsiddarthreddy** [comment]
Removed the db.sqlite3 file from the commit.

**Pull request merged**